frame/examples/default-config/src/lib.rs

```diff
@@ -107,17 +107,14 @@ pub mod pallet {
 pub mod tests {
 	use super::*;
 	use frame_support::derive_impl;
+	use sp_runtime::traits::ConstU64;
 
 	use super::pallet as pallet_default_config_example;
 
-	type UncheckedExtrinsic = frame_system::mocking::MockUncheckedExtrinsic<Test>;
 	type Block = frame_system::mocking::MockBlock<Test>;
```

Comment:
And this line being removed everywhere.

Reply:
`UncheckedExtrinsic` no longer needs to be passed to `construct_runtime!`. Since it was not being used elsewhere in most files, it could be removed.